android/app/src/main/java/org/fossasia/openevent/viewmodels/AboutFragmentViewModel.java

```diff
@@ -28,34 +28,28 @@
 
     public AboutFragmentViewModel() {
         realmRepo = RealmDataRepository.getDefaultInstance();
-        sessions = new MutableLiveData<>();
         dateList = new MutableLiveData<>();
-        eventLiveData = new MutableLiveData<>();
-        subscribeToEvent();
         subscribeToDateList();
     }
 
-    private void subscribeToSessions(ArrayList<String> dates) {
-        LiveRealmData<Session> sessionLiveRealmData = RealmDataRepository.asLiveData(realmRepo.getBookMarkedSessions());
-        sessions = Transformations.map(sessionLiveRealmData, input -> getSessionsList(dates, input));
-    }
-
     private void subscribeToDateList() {
         RealmResults<EventDates> eventDates = realmRepo.getEventDatesSync();
         ArrayList<String> dateListString = new ArrayList<>();
         for (EventDates eventDate : eventDates) {
             dateListString.add(eventDate.getDate());
         }
         dateList.setValue(dateListString);
-        subscribeToSessions(dateListString);
-    }
-
-    private void subscribeToEvent() {
-        LiveRealmDataObject<Event> liveRealmDataObject = RealmDataRepository.asLiveDataForObject(realmRepo.getEvent());
-        eventLiveData = Transformations.map(liveRealmDataObject, input -> input);
+        if (sessions == null) {
+            LiveRealmData<Session> sessionLiveRealmData = RealmDataRepository.asLiveData(realmRepo.getBookMarkedSessions());
+            sessions = Transformations.map(sessionLiveRealmData, input -> getSessionsList(dateListString, input));
+        }
```

Comment:
Similar should be done for dateList as well

Reply:
Please review now